andela/mmdp_site
# MMDP-290: User change password api

**What does this PR do?**

Allows user to change their password by providing their current password, new password and confirm password.

**description of Task to be completed?**

- [x] Allow users to be able to change their password.

**How should this be manually tested?**

- `git fetch`
- `git checkout ft-change-password-api-MMDP-290`
- `yarn run dev-server`
- input this link `<your-cms-domain>api/v1/users/changePassword` into postman and provide your old password, new password and confirm password.
- use a PUT method.

**What are the relevant Jira issues?**

[MMDP-290](https://viisaus.atlassian.net/browse/MMDP-290)

**Screenshots (if appropriate)**

![image](https://user-images.githubusercontent.com/27225249/57675782-9bbc7000-762b-11e9-8fb2-9a4a35018c1b.png)

**Questions:**

None
